contrib/seeds/README.md

```diff
@@ -8,13 +8,17 @@ and remove old versions as necessary (at a minimum when SeedsServiceFlags()
 changes its default return value, as those are the services which seeds are added
 to addrman with).
 
-The seeds compiled into the release are created from sipa's DNS seed and AS map
+The seeds compiled into the release are created from sipa's and achow101's DNS seed and AS map
 data. Run the following commands from the `/contrib/seeds` directory:
 
 ```
 curl https://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
+curl https://mainnet.achownodes.xyz/seeds.txt.gz | gzip -dc >> seeds_main.txt
+curl https://testnet.achownodes.xyz/seeds.txt.gz | gzip -dc > seeds_test.txt
 curl https://bitcoin.sipa.be/asmap-filled.dat > asmap-filled.dat
```

Comment:
How up-to-date is that ASMap file? If it's not up updated regularly it might be better to use one of the ones available at https://github.com/fjahr/asmap-data. Maybe this is a change in process that requires more eyes on it and I wouldn't call this a blocker then but I would like to propose this as a goal for the next release to use something more recent.

cc: @sipa

Reply:
Yes, if it's old, we should use a more up to date one. Can that repo somehow have a permalink to the most recent asmap? Otherwise we would run into the same problem with outdated asmaps.

Reply:
> Can that repo somehow have a permalink to the most recent asmap? Otherwise we would run into the same problem with outdated asmaps.

I hadn't really thought about this so far. This is a bit hacky but for now I copied the latest asmap file there to `latest_asmap.dat` and I would update that file as we create new maps so [this link](https://github.com/fjahr/asmap-data/raw/main/latest_asmap.dat) should always point to the latest file. I will look into making this a bit nicer by having latest.asmap.org link to that file automatically but that shouldn't block this PR here. I can update this as a follow-up.

Reply:
Changed to use that asmap, although did not regenerate the seeds.

Reply:
> I think you can make a symlink in the repo and just make sure it's updated every time.

I tried this but it seems like it's not supported. I didn't an official statement but here is a discussion that seems to confirm there is not way to use symlinks for this: https://github.com/dear-github/dear-github/issues/156